site/public/ts/sql-toolbox.js

```diff
@@ -1,4 +1,4 @@
-import { buildCourseUrl, getCsrfToken } from './server.js';
+import { buildCourseUrl, getCsrfToken } from './server.ts';
```

Comment:
Is the file extension necessary?

Reply:
jest was complaining when it was on `.js` but yeah not needed so removed